### ISSUE TITLE
How to build on windows?

### ISSUE THREAD

**Issue by xiaohenian** [amiremohamadi/DuckX#17]
How to build on windows?

**Comment by amiremohamadi** [comment]
Hi @xiaohenian.
**Duckx** requires some libraries, including **libzip**, so you must install it first!
If libzip is installed, you can add **Duckx resources** to your project and compile them! :beers: :smiley:

I know this process is a little bit boring.
I've been working on this for some time, and a quicker way to use duckx will soon be available.

**Comment by xiaohenian** [comment]
Thanks。Can this library create, modify, and save word documents?

**Comment by amiremohamadi** [comment]
Yes! you can save the document but some features are not completed yet.
Currently, Duckx is under development and the first release will be released next month.

**Comment by xiaohenian** [comment]
This means that Duckx can now create and save documents, but can't modify the document?

**Comment by amiremohamadi** [comment]
Already yes.You can read and create document.
Any suggestion and contribution is welcome.
Thank You! :smiley:
